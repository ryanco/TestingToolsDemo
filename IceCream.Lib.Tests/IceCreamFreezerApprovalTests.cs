using ApprovalTests;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace IceCream.Lib.Tests
{
	[TestClass]
	public class IceCreamFreezerApprovalTests
	{
		[TestMethod]
		public void AddToFreezerApprovalTest()
		{
			var freezer = new IceCreamFreezer();
			freezer.AddIceCream(new IceCream { Name = "Tastey" });
			freezer.AddIceCream(new IceCream { Name = "Lite" });

			Approvals.VerifyAll("Freezer Contents Test:", freezer.Contents(), "Ice Cream");
		}
	}
}
