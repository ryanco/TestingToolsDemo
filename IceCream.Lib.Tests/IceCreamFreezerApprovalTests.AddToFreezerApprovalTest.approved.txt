Freezer Contents Test:

Ice Cream[0] = Name: Tastey Flavor:  Cost: 0
Ice Cream[1] = Name: Lite Flavor:  Cost: 0
